raiden/api/rest.py

```diff
@@ -1173,6 +1174,8 @@ def _withdraw(
             return api_error(errors=str(e), status_code=HTTPStatus.BAD_REQUEST)
         except (InsufficientFunds, WithdrawMismatch) as e:
             return api_error(errors=str(e), status_code=HTTPStatus.CONFLICT)
+        except InsufficientEth as e:
```

Comment:
Withdraw will trigger the Raiden service to dispatch the state change into the state machine, wait for the partner's confirmation and then the `ContractSendChannelWithdraw` would be handled in a separate greenlet. This means that the exception taking place while performing on-chain channel withdraw will not be propagated here.

Reply:
Thanks for pointing this out, I will move it to the correct place.